app/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
     <string name="action_ok">Got it!</string>
     <string name="action_apply_theme">Apply Theme</string>
     <string name="action_change_memory_location">Change location</string>
+    <string name="action_change_memory_marker_restaurant"></string>
```

Comment:
This string resource does not have a value and isn't used anywhere. It should be removed.

Reply:
oki